app/views/mailers/reservation_mailer/car_reservation_cancel_student.html.erb

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+  <h3>Your transportation reservation has been canceled. </h3>
+    <p>
+     Reservation Summary:
+      <ul>
+        <li>DATE: <%= "#{@start_time} - #{@end_time}" %></li>
+        <li>VEHICLE #: <%= link_to @reservation.car.car_number, car_url(@reservation.car) %></li>
+        
+        <li>SITE: <%= link_to @reservation.site.title, site_url(@reservation.site) %> </li>
+        <li>PROGRAM: <%= link_to @reservation.program.title, program_url(@reservation.program) %></li>
+        <li>DRIVER: <%= @driver_name %> </li>
+        <li>BACKUP DRIVER: <%= @backup_driver_name %> </li>
+      </ul>
+      PASSENGERS:
+      <ul>
+        <% @passengers.each do |p| %>
+          <li> <%= p %> </li>
+        <% end %>
+      </ul>
+    </p>
+    <p>
+      You can email psych.transportation@umich.edu or call the office at: 734-764-2580.
```

Comment:
Again using unit_preference variables and not static value

Reply:
done in [37fb120](https://github.com/lsa-mis/lsa_ride_share/pull/160/commits/37fb1207bf4b24c6abf96c8892dd6efc45e5cff7)